html/includes/api_functions.inc.php

```diff
@@ -1513,3 +1513,52 @@ function validate_column_list($columns, $tableName)
         $app->stop();
     }
 }
+
+function add_service_for_host()
+{
+    check_is_admin();
```

Comment:
We should also allow normal users to add these if they are allowed to access the device.

You can use `check_device_permission($device_id);` once you have the device id below.

Reply:
Changed in 7344482